internal/controller/authenticator.go

```diff
@@ -115,7 +118,7 @@ func (authn *Authenticator) WriteK8sSecret(ctx context.Context, keyData OnboardR
 	return nil
 }
 
-// NewClientSet returns a new Kubernetes clientset based on the execution environment.
+// Update NewClientSet to use the helper function
```

Comment:
What does this comment mean?
Why replace a comment that follows the Golang style guide with one that doesn't?